railties/lib/rails/configuration.rb

```diff
@@ -64,9 +65,10 @@ def unshift(*args, &block)
       end
 
       def merge_into(other) #:nodoc:
-        @operations.each do |operation, args, block|
+        @operations.concat(@delete_operations).each do |operation, args, block|
```

Comment:
Ah ok. So the use case is when merge_into has been called and we call it again which means new operations are appended after delete operation?